tuplex/core/src/ee/local/LocalBackend.cc

```diff
@@ -898,13 +994,13 @@ namespace tuplex {
                     lines.push_back(Row((int64_t)opid, exceptionCodeToPythonClass(ec), (int64_t)keyval.second));
                 }
             }
-            // input partitions given?
+
             if(!tstage->inputExceptions().empty()) {
-                size_t numSlowPath = 0;
-                for(auto& p : tstage->inputExceptions())
-                    numSlowPath += p->getNumRows();
-                lines.push_back(Row("(cached)", exceptionCodeToPythonClass(ExceptionCode::NORMALCASEVIOLATION), (int64_t)numSlowPath));
-                totalECountsBeforeResolution += numSlowPath;
+                size_t numExceptions = 0;
+                for (auto &p : tstage->inputExceptions())
+                    numExceptions += p->getNumRows();
+                lines.push_back(Row("(schema)", exceptionCodeToPythonClass(ExceptionCode::NORMALCASEVIOLATION), (int64_t)numExceptions));
```

Comment:
what's the reason for the change to (schema) ? Trying to understand

Reply:
The problem was that there is now not a difference between generalCase exceptions that are saved when a pipeline result is cached and python objects that occur during parallelize. So there's no telling whether an `inputException` was from `cache` or `parallelize` so I thought to generalize it to something like "(schema)" or maybe "(input)"

Reply:
I see, makes sense. Let's use schema then.